core/src/main/resources/schema/app/20220303125000_CB-16079_alter_stackpatch_table_add_status_columns.sql

```diff
@@ -0,0 +1,15 @@
+-- // CB-16079
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stackpatch ADD COLUMN IF NOT EXISTS status CHARACTER VARYING(255);
+UPDATE stackpatch SET status = 'FIXED' WHERE status IS NULL;
```

Comment:
Can it cause any issue that the undo script does not have a "revert" for this update?

Reply:
You are right, the revert should remove all rows that do not have FIXED status